code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -130,3 +130,8 @@
 		for(var/obj/item/weapon/bomberman/dispenser in src)
 			if(dispenser.spam_bomb)
 				dispenser.attack_self(src)
+
+		if(get_active_hand() && istype(get_active_hand(),/obj/item/weapon/rcl))

```

Comment:
I was actually planning a system based on a STEP_ACTION flag and a mob list, but the inventory code isn't really consistent enough to try that. This also has some very unnecessary proc calls, since get_active_hand is called 3 times more than necessary.